src/vs/workbench/contrib/extensions/electron-browser/extensionsActions.ts

```diff
@@ -1753,6 +1753,75 @@ export class ShowRecommendedExtensionsAction extends Action {
 	}
 }
 
+export class ShowAppLaunchRecommendedExtensionsAction extends Action {
```

Comment:
2 things:
- Don't these need to be in a SQL Carbon Edit tag section?
- Why not put these under the src/sql folder and just import in when referencing elsewhere? I don't see a reason these need to be in the src/vs tree as it will add merge issues.